### DISCUSSION TITLE
Jumper wire in different configuration of the ethernet module

### DISCUSSION THREAD

**Discussion by mDubovsky** [pilotak/eth-io#8] [Ideas]
Some compatible ethernet modules have the oscillator on the top side of the PCB.
We should add photo of the jumper wire in this configuration:
![image](https://user-images.githubusercontent.com/9304179/208427753-92d21ac7-5e7a-4eb8-8f17-4317a87dfc32.png)

**Edit: this is not my photo and am not sure of the copyright on this one. I can make a similar photo as I do have a module like this.**

**Comment by pilotak** [comment]
The problem i had with this top side version is that it's a little longer and the mouting holes do not fit properly

**Reply by mDubovsky** [reply]
Maybe we have a bit different versions? I have a "top side version" and both the footprint and the mounting holes fit perfectly. 

![image](https://user-images.githubusercontent.com/9304179/208465275-0b5eeacd-a5c7-484e-99aa-238425a9488c.png)

**Reply by pilotak** [reply]
![eth](https://user-images.githubusercontent.com/8101753/208469693-5176fa73-8e99-4477-a355-2da79696dbe7.jpg)

**Reply by mDubovsky** [reply]
So that is a different version you have. I would still include the photo with the version that I have along with a word of warning that the version that you have doesn't flip properly so people don't mix those up.

**Reply by pilotak** [reply]
I agree, please submit a PR
